From 0a73f81f2e0ea8808ae94e5d5a2dd1f86ffa52a6 Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Mon, 31 Mar 2014 16:36:27 +0100
Subject: [PATCH 286/351] mfd: florida: Remove non-existent interrupts

The DCS_DAC_DONE and DCS_HP_DONE interrupts do not exist on the Florida
version of the IP. This patch removes them from the regmap IRQ
structure.

Change-Id: I94a21c06daad62cc687f882264cf8074b35eb86e
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
---
 drivers/mfd/florida-tables.c |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/mfd/florida-tables.c b/drivers/mfd/florida-tables.c
index 69713e5..249bc74 100644
--- a/drivers/mfd/florida-tables.c
+++ b/drivers/mfd/florida-tables.c
@@ -420,12 +420,6 @@ static const struct regmap_irq florida_irqs[ARIZONA_NUM_IRQ] = {
 	[ARIZONA_IRQ_BOOT_DONE] = {
 		.reg_offset = 4, .mask = ARIZONA_BOOT_DONE_EINT1
 	},
-	[ARIZONA_IRQ_DCS_DAC_DONE] = {
-		.reg_offset = 4, .mask = ARIZONA_DCS_DAC_DONE_EINT1
-	},
-	[ARIZONA_IRQ_DCS_HP_DONE] = {
-		.reg_offset = 4, .mask = ARIZONA_DCS_HP_DONE_EINT1
-	},
 	[ARIZONA_IRQ_FLL2_CLOCK_OK] = {
 		.reg_offset = 4, .mask = ARIZONA_FLL2_CLOCK_OK_EINT1
 	},
-- 
1.7.9.5

